### PR TITLE
Disable LanguageConstructDebugTest, ControlFlowDebugTest, ModuleExecutionFlowTests and LogAPITestCase.testAllOn tests

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will disable LanguageConstructDebugTest, ControlFlowDebugTest, ModuleExecutionFlowTests and LogAPITestCase.testAllOn tests.

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ public void testTraceLevel() throws BallerinaTestException {
         validateLog(logLines[9], "TRACE", "[]", traceLog);
     }
 
-    @Test(description = "Tests log functionality when all log levels are turned on")
+    @Test(description = "Tests log functionality when all log levels are turned on", enabled = false)
```
Comment (MaryamZi) [inline tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java:201]: A different test seems to be failing now

`org.ballerinalang.test.logging.LogAPITestCase > testBasicLogFunctionality`

We may have to disable more than just this.

File: tests/jballerina-unit-test/src/test/resources/testng-new-parser.xml [hunk 2]
```diff
@@ -130,7 +130,8 @@
             <package name="org.ballerinalang.test.taintchecking.*"/>
             <package name="org.ballerinalang.test.functions.*"/>
             <package name="org.ballerinalang.test.javainterop.*"/>
-            <package name="org.ballerinalang.test.execution.*" />
+            <package name="org.ballerinalang.test.execution.LargeInitBuildTest" />
+<!--            <package name="org.ballerinalang.test.execution.ModuleExecutionFlowTests" />-->
```
Comment (MaryamZi) [inline tests/jballerina-unit-test/src/test/resources/testng-new-parser.xml:134]: This is still failing for the old parser test suite. Can you please update `testng-jballerina.xml` also?

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java (hunk 1 above)
Comment (praveennadarajah) [reply tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java:201]: If that is the case shall we disable all the test cases in LogAPITestCase class?

File: tests/jballerina-unit-test/src/test/resources/testng-new-parser.xml (hunk 2 above)
Comment (praveennadarajah) [reply tests/jballerina-unit-test/src/test/resources/testng-new-parser.xml:134]: Disabled in `testng-jballerina.xml`

File: tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java (hunk 1 above)
Comment (praveennadarajah) [reply tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java]: Disabled all the test cases in `LogAPITestCase` class.

Comment (MaryamZi) [reply tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java]: @MadhukaHarith92, please note.

Comment (MadhukaHarith92) [reply tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/logging/LogAPITestCase.java]: Ack.

**Review by MaryamZi — APPROVED** [review]
Approving since all the other PRs are currently blocked because these tests are failing.

`LogAPITestCase` and `ModuleExecutionFlowTests` pass locally, and we suspect the GitHub action builds are failing due to extra output `Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8`.

**Comment by MadhukaHarith92** [comment]
If https://github.com/ballerina-platform/ballerina-lang/pull/25349 resolves the build issue, it won't be necessary to disable tests

**Comment by praveennadarajah** [comment]
#25349 resolves the issue. Therefore closing it.

**Pull request closed without merging**